cadc-util/src/main/java/ca/nrc/cadc/auth/SSOCookieManager.java

```diff
@@ -161,28 +230,20 @@ public void setOffsetExpiryHours(int offsetExpiryHours)
      *
      * @param cookieValue
      * @param requestedDomain
-     * @param expiryDate
      * @return cookieList
      */
-     public List<SSOCookieCredential> getSSOCookieCredentials(final String cookieValue, final String requestedDomain, final Date expiryDate) {
-        List<SSOCookieCredential> cookieList = new ArrayList<>();
-
-        // Make cookie with requested domain
-        SSOCookieCredential requestedCookie = new SSOCookieCredential(cookieValue, requestedDomain, expiryDate);
-        cookieList.add(requestedCookie);
+    public List<SSOCookieCredential> getSSOCookieCredentials(final String cookieValue, final String requestedDomain)
```

Comment:
I find the requestedDomain argument here misleading...

Reply:
agreed. done